Libplanet/Net/Swarm.cs

```diff
@@ -1891,6 +1891,16 @@ await foreach (Block<T> block in blocks)
                             block.Hash
                         );
                     }
+
+                    var getBlockListCount = await blocks.CountAsync(cancellationToken);
```

Comment:
I guess we can use a separated counter manually instead of  `.CountAsync()`. (or, just save `receivedBlockCount` before `GetBlocksAsync()`)

Reply:
amended at https://github.com/planetarium/libplanet/pull/1112/commits/969588c050c433949763a7ec7fee88f9f25e7cfa